mvc/views/razor.rst

```diff
@@ -1,8 +1,283 @@
-.. include:: /../common/stub-topic.txt
+Razor Syntax In MVC 6
+=====================
+By `John Rowley`_
 
-|stub-icon| Razor Syntax
-========================
+---------------
+What is Razor?
+---------------
+Razor is the ASP.NET view engine used in MVC. It allows c# and
```

Comment:
Perhaps use the line from readme instead:

> The Razor syntax provides a fast, terse, clean and lightweight way to combine server code with HTML to create dynamic web content. This repo contains the parser and the C# code generator for the Razor syntax.

Razor isn't a view engine, we just happen to build a view engine that parses and compiles files using the Razor syntax.